integrations/hello-world-ng5/package.json

```diff
@@ -35,6 +35,6 @@
     "npx": "^10.2.0",
     "ts-node": "4.1.0",
     "tslint": "5.9.1",
-    "typescript": "2.7.1"
+    "typescript": "2.8.1"
```

Comment:
Do you remember which feature required this upgrade?

Reply:
Was it the `infer`?

Reply:
Yes, `infer`.  In fact, if in the current integration ng5 test we add the use of state operators `patch(), ...` the build will fall as it needs TS 2.8

Reply:
Yeah, that is part of the reason why I made that a subpackage so that the main lib was not forced to upgrade.